authors/apps/comments/views.py

```diff
@@ -0,0 +1,142 @@
+from django.shortcuts import render
+from rest_framework import status, exceptions
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from django.shortcuts import get_object_or_404
+
+from authors.apps.articles.models import Article
+from .models import Comment
+
+from .serializers import CommentSerializer, RecursiveField
+from .renderers import CommentJSONRenderer
+
+from django.core.serializers.json import DjangoJSONEncoder
+import json
+from drf_yasg.utils import swagger_auto_schema
+
+
+class CommentAPIView(APIView):
+    permission_classes = (IsAuthenticated,)
+    renderer_classes = (CommentJSONRenderer,)
+    serializer_class = CommentSerializer
+
+    @swagger_auto_schema(
+        operation_description='Create a Comment.',
+        operation_id='Comment on an article',
+        request_body=serializer_class,
+        responses={201: serializer_class(many=False), 400: 'BAD REQUEST'},
+    )
+    def post(self, request, *args, **kwargs):
+        comment = request.data.get('comment', {})
+        slug = kwargs.get('slug')
+        parent = request.data.get('comment', None).get('parent_id', None)
```

Comment:
> This looks like repitition since you already got the comment on line 31

@cdvx I was getting the parent independently because it's needed by the `RecursiveField` serializer. I think let me change that line to `parent = comment.get('parent_id'), None)`